src/services/cleanFormSubmissionModel.ts

```diff
@@ -210,6 +218,15 @@ function cleanElementValue(
         }
         break
       }
+      case 'text':
+      case 'textarea':
+      case 'email':
+      case 'barcodeScanner':
+      case 'telephone': {
+        const value = submission[element.name] as string
+        model[element.name] = value.trim()
+        break
+      }
```

Comment:
This will prevent the conditional logic stuff below (in the 'default' clause) from running on these element types. We need to do this stuff inside that default so all that logic still runs